src/axom/mir/docs/sphinx/mir_clipping.rst

```diff
@@ -0,0 +1,137 @@
+|                                 | of the field that is used for clipping. At present,  |
+|                                 | the field must be a vertex-associated field.         |
++---------------------------------+------------------------------------------------------+
+| ``clipValue: value``            | An optional numeric argument that specifies the      |
+|                                 | value in the field at which the clip boundary is     |
+|                                 | defined. The default is 0.                           |
++---------------------------------+------------------------------------------------------+
+| ``colorField: name``            | If inside=1 and outside=1 then a color field is      |
+|                                 | generated so it is possible to tell apart regions of |
+|                                 | the clip output that were inside or outside the clip |
+|                                 | boundary. This field permits the user to change the  |
+|                                 | name of the color field, which is called "color" by  |
+|                                 | default.                                             |
++---------------------------------+------------------------------------------------------+
+| ``coordsetName: name``          | The name of the new coordset in the output mesh. If  |
+|                                 | it is not provided, the output coordset will have the|
+|                                 | same name as the input coordset.                     |
++---------------------------------+------------------------------------------------------+
+|``fields:``                      | The fields node lets the caller provide a list of    |
+|                                 | field names that will be processed and added to the  |
+|                                 | output mesh. The form is *currentName:newName*. If   |
+|                                 | the *fields* node is not given, the algorithm will   |
+|                                 | process all input fields. If the fields node is empty|
+|                                 | then no fields will be processed.                    |
++---------------------------------+------------------------------------------------------+
+| ``inside: number``              | Indicates to the clipping algorithm that it should   |
+|                                 | preserve zone fragments that were "inside" the clip  |
+|                                 | boundary. Set to 1 to enable, 0 to disable. The      |
+|                                 | algorithm will generate these fragments by default.  |
++---------------------------------+------------------------------------------------------+
+| ``originalElementsField: name`` | The name of the field in which to store the original |
+|                                 | elements map.                                        |
```

Comment:
I think this is the only one without the default specified.

Reply:
Fixed.